homeassistant/components/airvisual/strings.sensor.json

```diff
@@ -0,0 +1,20 @@
+{
+  "state": {
+    "airvisual__pollutant_label": {
+      "co": "Carbon Monoxide",
+      "n2": "Nitrogen Dioxide",
+      "o3": "Ozone",
+      "p1": "PM10",
+      "p2": "PM2.5",
+      "s2": "Sulfur Dioxide"
```

Comment:
I'm consufed these look like sensor descriptions, but are sensor states?

Reply:
Correct: this is the "What is the current primary (most prevalent) pollutant?" sensor.